docs/guides/advanced/data/4844_data.md

```diff
@@ -0,0 +1,185 @@
+# Understanding zkSync's Use of 4844 Blob Transactions
+
+This article explains how zkSync is utilizing a new type of transaction called 4844 blob transactions to transfer its
+public data (pubdata) to Ethereum Layer 1 (L1).
+
+We will skip the technical details of 4844 blobs and focus on how zkSync specifically implements them.
+
+## Before 4844: The Era of Calldata
+
+Originally, before the introduction of 4844 blobs, the system worked differently. In this period, known as the pre-4844
+era, zkSync would collect all its pubdata, which could be up to 120kb. This data was then sent as 'calldata' of the
+commitBatches transaction.
+
+In these transactions, the contract would calculate a keccak hash of the data and use this hash in the proving process.
+More precisely, this hash was stored as part of the block commitment and later used in the `proveBatches` function.
```

Comment:
The pubdata was not used in the proving process.

in the proving process we only used the hash of all the state diffs (i.e. uncompressed ones). This hash is calculated on L2 and passed by system logs. It will continue being the case for the blob based system too.

Only together with blob introduction the commitments to the chunks of pubdata will be needed to prove the blob correctness

Reply:
only the hash of the state diffs was part of batch commitment (i.e. the data that is used by prover)

the hash of the pubdata was published via system L2->L1 logs and accessed during `commitBatches` function call & it was there to make sure that the correct pubdata has been sent